release.json

```diff
@@ -25,12 +25,12 @@
         "WINDOWS_APMINJECT_SHASUM": "27d85ab3a26c123b2655a838b0bec099268de2f2b86d2b8a74232e65f4f8f05f"
     },
     "nightly-a7": {
-        "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "dee9118895ef57c93650f421015d88872e6bae1f",
-        "OMNIBUS_RUBY_VERSION": "f3fc847e03ba7081e266b2d333210ba129128a14",
+        "INTEGRATIONS_CORE_VERSION": "7.56.x",
+        "OMNIBUS_SOFTWARE_VERSION": "7.56.x",
```

Comment:
This feels wrong as it switches from a pinned commit to a moving target, shouldn't it be a tag, or the commit on top of the branch instead?

Reply:
To clarify the implication, this would prevent omnibus git cache from detecting a change, and it would likely use a stale cache instead of regenerating it